.slather.yml

```diff
@@ -5,4 +5,9 @@ ignore:
   - ../Library/*
   - ../../Library/*
   - ../../../Library/*
+  - "**/Vendor/DatadogCore/*"
+  - "**/Vendor/DatadogInternal/*"
+  - "**/Vendor/DatadogLogs/*"
+  - "DatadogPrivate-Objc/*"
+  - "**/DatadogPrivate-Objc/*"
   - Tests/*
```

Comment:
I find it slightly ironic that we ignore the `Tests/` directory for coverage. Am I missing something? I know its not part of this PR but noticed anyhow

Reply:
If you didn't include these, we would need to write tests to cover our tests...

Reply:
We don't test our unit tests so we omit these from the coverage check

Reply:
OOOOOO that makese sense. For some reason I thought it had to do with don't run any of the tests under `Test/` for coverage purposes. Got it - we definitely don't want to write tests for our tests 😂. Thanks for the clarity all!